### DISCUSSION TITLE
Ubuntu distr and installation

### DISCUSSION THREAD

**Discussion by Salvodif** [bpozdena/OneDriveGUI#209] [Ideas]
Hi,

I know that having an appimage is easier but it's really bad to see on Ubuntu not having the programme installed and having to call up the downloaded file.
Wouldn't it be possible to have an installable programme?

**Comment by bpozdena** [comment]
Sorry, AppImage is the only package I'm willing to maintain myself. It will work on any distro. There are many tools that will help you integrate/install AppImages into your desktop environment.  You can try AppImageLauncher or Gear Lever .
